src/Designer/frontend/libs/studio-assistant/src/components/MessageColumn/Messages/AssistantMessage/SourceList/SourceList.module.css

```
@@ -0,0 +1,155 @@
+  justify-content: space-between;
+  margin-bottom: var(--ds-size-2);
+  gap: var(--ds-size-2);
+}
+
+.sourceTitle {
+  font-size: var(--ds-font-size-small);
+  font-weight: 600;
+  color: var(--ds-color-text-primary);
+  text-decoration: none;
+  word-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)